src/payment/data/service.js

```diff
@@ -3,12 +3,68 @@ import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 
 import handleRequestError from './handleRequestError';
 import { transformResults } from './utils';
+import { isWaffleFlagEnabled } from '../../data/waffleFlags';
+import { WAFFLE_FLAGS } from './constants';
 
 ensureConfig([
   'ECOMMERCE_BASE_URL',
   'LMS_BASE_URL',
 ], 'payment API service');
 
+/* eslint-disable no-use-before-define */ /* Chicken vs Egg...
+    but it is preferred the interpreter pickup errors rather than use magic strings */
+
+/**
+ * Ecommerce IDA/Coordinator IDA URLs keyed by the function name they are meant for.
+ *
+ * @type {{[p: string]: string}}
+ */
+const urlsByFunction = {
```

Comment:
It sounds like this will require filing tickets for remapping the Coordinator & API specs to use similar endpoints names as Ecommerce?

What if we want to name something differently in the future?

Reply:
We did that in PR 100. The documentation is wrong now. Was your suggestion.